base_location/models/company.py

```diff
@@ -6,19 +6,46 @@
 
 
 class ResCompany(models.Model):
-
     _inherit = 'res.company'
 
-    @api.onchange('better_zip_id')
-    def on_change_city(self):
-        if self.better_zip_id:
-            self.zip = self.better_zip_id.name
-            self.city = self.better_zip_id.city
-            self.state_id = self.better_zip_id.state_id
-            self.country_id = self.better_zip_id.country_id
+    city_id = fields.Many2one(
```

Comment:
Why adding this field here? It's out of this module's scope.

Reply:
This comment still remains

Reply:
City_id is not added on `base_address_city` module. So I had to add it in order to maintain its logic.